include/openssl/ct.h

```diff
@@ -481,7 +485,7 @@ int i2o_SCT_signature(const SCT *sct, unsigned char **out);
 * |len| should be the length of the signature in |in|.
 * Returns the number of bytes parsed, or a negative integer if an error occurs.
 */
-int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
+__owur int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
```

Comment:
My preferred solution here is to extract the 4 fields from the SCT struct that are related to signatures and put them in their own struct. That would allow me to make these two functions behave like the others. I'd still be happy to rename them though (I'm no fan of the "o2i" naming, it's rather vague and cryptic).